media/rapps/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# RAPPS .txt files are distributed as part of ReactOS.
+*.txt text eol=crlf
```

Comment:
The files in media/drivers/etc/ may also need CRLF. Theya are included in system32/etc and are networking-related files.

Reply:
Good catch! Fixed in 9af14d9